src/widgets/Number/components/Number.js

```diff
@@ -1,19 +1,25 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import _round from 'lodash/round';
 import './Number.scss';
 
+const getRounded = decimal => _round(decimal, 2);
+const getChange = (current, previous) => (previous === 0 ? previous : (current - previous) / previous * 100); // eslint-disable-line no-mixed-operators
+const getChangeIcon = number => (number < 0 ? <i className="fa fa-arrow-down" /> : <i className="fa fa-arrow-up" />);
+
 const Number = ({ title, data }) =>
   <div className="Number panel">
     <div className="panel-heading">
       {title}
     </div>
     <div className="panel-body">
-      <span className="this-week number-big">
-        {data.current}
+      <span className="current">
+        {getRounded(data.current)}
       </span>
-      {data.previous &&
-        <span className="last-week number-normal">
-          / {data.previous}
+      {typeof data.previous !== 'undefined' &&
+        <span className="previous">
```

Comment:
We should change `className` since it's not `previous` anymore, it's percentage.